cmd/link/cmd.go

```diff
@@ -28,8 +28,8 @@ import (
 var Cmd = &cobra.Command{
 	Use:     "link",
 	Aliases: []string{"associate"},
-	Short:   "Link a ocm/user role from stdin",
-	Long:    "Link a ocm/user role from stdin",
+	Short:   "Link ocm role to specific OCM organization",
```

Comment:
@ciaranRoche We're using `ocm` and `OCM` in the same sentence. Which one is our preferred?

Reply:
Good catch, updated to `OCM`